src/amazon/dsstne/utils/Filters.cpp

```diff
@@ -190,62 +176,49 @@ void SamplesFilter::loadFilter(unordered_map<string, unsigned int>& xMInput,
      Filter
     */
 
-
-    samplefilters.reset(new vector<unique_ptr<unordered_map<int,float>>>(xMSamples.size()));
+    samplefilters.reset(new vector<unique_ptr<unordered_map<int, float>>>(xMSamples.size()));
 
     vector<string> files;
-    if (listFiles(filterFilePath, false, files) == 0) {
+    if (listFiles(filterFilePath, false, files) == 0)
+    {
         cout << "Loading " << files.size() << " filter files" << endl;
 
-        for (auto const &file: files) {
+        for (auto const &file : files)
+        {
             cout << "\tLoading filter: " << file << endl;
             loadSingleFilter(xMInput, xMSamples, *samplefilters.get(), file);
         }
     }
 
-    cout << "Info:SamplesFilter " << samplefilters->size() <<endl;
+    cout << "Info:SamplesFilter " << samplefilters->size() << endl;
 }
 
-void SamplesFilter::applyFilter(float *xArray,int xSamplesIndex, int offSet, int width)
+void SamplesFilter::applyFilter(float *xArray, int xSamplesIndex, int offset, int width)
 {
-    unordered_map<int,float> *filter= (*samplefilters)[xSamplesIndex].get();
-    updateRecords(xArray,filter, offSet, width);
+    unordered_map<int, float> *filter = (*samplefilters)[xSamplesIndex].get();
+    updateRecords(xArray, filter, offset, width);
 }
 
-void SamplesFilter::applyFilter(float *xArray,int xSamplesIndex)
+void SamplesFilter::applyFilter(float *xArray, int xSamplesIndex)
 {
-    unordered_map<int,float> *filter= (*samplefilters)[xSamplesIndex].get();
-    updateRecords(xArray,filter);
+    unordered_map<int, float> *filter = (*samplefilters)[xSamplesIndex].get();
+    updateRecords(xArray, filter);
 }
 
-SamplesFilter::~SamplesFilter()
+FilterConfig* loadFilters(const std::string &samplesFilterFileName,
```

Comment:
minor: there is an inconsistency of & usage, in one case it is on the right side (const std::string &outputFileName) on other on the left (unordered_map<string, unsigned int>& xMInput)